src/main/java/com/github/lipinskipawel/board/engine/LogicalPoints.java

```diff
@@ -66,6 +66,13 @@ private int computeBallPosition(final Direction destination) {
      * @return
      */
     boolean isOtherPlayerToMove() {
+        if (ballPosition.getPosition() == 3 ||
+                ballPosition.getPosition() == 4 ||
+                ballPosition.getPosition() == 5 ||
+                ballPosition.getPosition() == 111 ||
+                ballPosition.getPosition() == 112 ||
+                ballPosition.getPosition() == 113)
+            return true;
         return ballPosition.getUnavailableDirection().size() == 1 || ballPosition.getAllowedDirection().size() == 8;
```

Comment:
you can return here one conditional expression and put 
`ballPosition.getPosition() == 3 ||
                ballPosition.getPosition() == 4 ||
                ballPosition.getPosition() == 5` 
and 
`ballPosition.getPosition() == 111 ||
                ballPosition.getPosition() == 112 ||
                ballPosition.getPosition() == 113`
 into ballPosition methods